index.html

```diff
@@ -2180,7 +2180,7 @@ <h4 class="modal-title" id="myModalLabel">Long Vesion Code of Conduct</h4>
                <div class="footer-social-icons">
                    <a href="http://bit.ly/pydelhi-git" target="_blank"><i class="fa-2x fa fa-github"></i></a>
                    <a href="http://bit.ly/pydelhi-fb" target="_blank"><i class="fa-2x fa fa-facebook"></i></a>
-                   <a href="http://bit.ly/pydelhi-twitter" target="_blank"><i class="fa-2x fa fa-twitter"></i></a>
+                   <a href="https://twitter.com/PyDelhiConf" target="_blank"><i class="fa-2x fa fa-twitter"></i></a>
```

Comment:
Can you please have a bit.ly link here so that we can track how many people visited it. It's better to ask @theskumar because he has created all links previously. This is to ensure that one person has access to all the links generated :)